src/defines/defines-creator.ts

```diff
@@ -0,0 +1,27 @@
+import {IDefines} from './defines';
+import {IDefine} from './define';
+import {MagickFormat} from '../magick-format';
+import {MagickDefine} from './magick-define';
+
+export abstract class DefinesCreator implements IDefines {
+  protected format: MagickFormat;
```

Comment:
I changed `format` from `public` to `protected` like in Magick.NET because it will be used in `WriteDefinesCreator` (it's used in Magick.NET)

Reply:
Thanks for fixing that.